mms/monitoring/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+sleep 60
+cd /opt/prometheus-2.0.0.linux-amd64
+./prometheus --config.file=prometheus.yml --storage.tsdb.path /opt/prometheus-2.0.0.linux-amd64/mydata& 
+service grafana-server start;
+cd /opt/scylla-grafana-monitoring/;
+./load-grafana.sh -p 127.0.0.1:9090 -g 3000 -a admin -v 2.0&
+sleep infinity
```

Comment:
@rusher81572 why not using https://github.com/scylladb/scylla-grafana-monitoring/blob/master/start-all.sh ?
We are constantly changing the monitoring stack and this specific usage will probably break soon.

Reply:
@tzach It looks like start-all.sh manages the docker container and does nothing inside the container. The point of this was to automate Scylla Monitoring with a 1 stop shop for MMS. If load-grafana.sh changes in the future, we can adjust it according to what the official docs say.

Reply:
We need the example to use start-all, as any user will do
The motivation is not just making it easy for users running MMS< but also educate them on Scylla standard tooling

Reply:
Do you recommend we copy the repo files over to MMS so we can automate the configs or have them clone the regular repo. The latter will be more difficult for the user.

Reply:
They should clone, or the example will quickly become out of date when the monitoring stack moves forward.
Another alternative is having the monitoring as a git submodule under MMS, not sure it worth the effort, but we can try.